contrib/format-excel/src/main/java/org/apache/drill/exec/store/excel/ExcelBatchReader.java

```diff
@@ -412,7 +413,7 @@ private void addColumnToArray(TupleWriter rowWriter, String name, TypeProtos.Min
 
   /**
    * Returns the index of the first row of actual data.  This function is to be used to find the header row as the POI skips blank rows.
-   * @return:  The headerRow index, corrected for blank rows
+   * @return int The headerRow index, corrected for blank rows
```

Comment:
nit: there is no need to specify return type in `@return`, it looks awkward a little bit.

Reply:
Fixed